api/src/wfl/module/covid.clj

```diff
@@ -25,20 +25,20 @@
 ;; interfaces
```

Comment:
Many / most of these multimethods moved FYI:

https://github.com/broadinstitute/wfl/blob/develop/api/src/wfl/stage.clj

Reply:
Don't worry about it.  I'll just lint again before merging.